infra/feast-operator/config/samples/v1alpha1_featurestore_db_persistence.yaml

```
@@ -11,5 +11,25 @@ spec:
         store:
           type: postgres
           secretRef:
-            name: my-secret
-          secretKeyName: mykey   # optional
+            name: postgres-secret
+          secretKeyName: postgres-secret-parameters   # optional
+    registry:
+      local:
+        persistence:
+          store:
+            type: sql
+            secretRef:
+              name: postgres-secret
+            secretKeyName: postgres-secret-parameters   # optional
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-secret
+stringData:
+  postgres-secret-parameters: |
+    path: postgresql://postgres:mysecretpassword@127.0.0.1:55001/feast
+    cache_ttl_seconds: 60
+    sqlalchemy_config_kwargs:
+      echo: false
+      pool_pre_ping: true
```

this sample wouldn't work without some changes -
```suggestion
            name: postgres-secret
    registry:
      local:
        persistence:
          store:
            type: sql
            secretRef:
              name: postgres-secret
            secretKeyName: postgres-secret-parameters   # optional
---
apiVersion: v1
kind: Secret
metadata:
  name: postgres-secret
stringData:
  postgres-secret-parameters: |
    path: postgresql+postgresql://postgres:mysecretpassword@127.0.0.1:55001/feast
    cache_ttl_seconds: 60
    sqlalchemy_config_kwargs:
      echo: false
      pool_pre_ping: true
  postgres: |
    host: 127.0.0.1
    port: 55001
    database: feast
    db_schema: public
    user: postgres
    password: mysecretpassword
```

---

please verify that this sample works in a cluster